templates/download/amd/tab-2.html

```diff
@@ -29,32 +29,34 @@ <h2 class="u-hide--small">
   <hr class="p-rule" />
   <div class="row p-strip u-no-padding--top">
     <div class="col-3 col-medium-2">
-      <h3>Ubuntu Desktop 22.04 LTS</h3>
+      <h3>Ubuntu Server 24.04</h3>
     </div>
     <div class="col-6 col-medium-3">
-      <p>The version of Ubuntu with up to 10 years of long term support, until April 2032.</p>
+      <p>The optimised Ubuntu Server 24.04 is beta for now, the certified version is coming soon.</p>
```

Comment:
```suggestion
      <p>The optimized Ubuntu Server 24.04 is beta for now, the certified version is coming soon.</p>
```
Should we update the spelling here since we are moving from UK to US spelling now

Reply:
I don't think so as the PM's new change had the UK version.